application/connection.mjs

```diff
@@ -0,0 +1,26 @@
+import { AUTH0_DOMAIN } from '#config';
+import getHeaders from '#utils/get-headers';
+
+export async function getConnectionByName(name) {
+  /**
+   *  https://auth0.com/docs/api/management/v2/connections/get-connections
+   */
+  const url = new URL(`https://${AUTH0_DOMAIN}/api/v2/connections`);
+  url.searchParams.set('name', name);
+
+  const response = await fetch(url, { headers: await getHeaders() });
+  const connections = await response.json();
+  return connections[0];
```

Comment:
Please ensure that Auth0 does not allow 2 connections with the same name.

Reply:
![image](https://github.com/THE-Engineering/sync-from-the-institutions-to-auth0-organizations/assets/60063071/31144b47-492c-4338-9b84-95208af5a5c7)